src/docs/asciidoc/deployment/configuration/web-ui_configuration.adoc

```diff
@@ -106,7 +106,7 @@ a|card time display mode in the feed. Values :
 |feed.geomap.initialLatitude|0|no| The initial latitude of the map when no cards with geographical coordinates are present.
 |feed.geomap.initialLongitude|0|no| The initial longitude of the map when no cards with geographical coordinates are present.
 |feed.geomap.initialZoom|1|no| Initial zoom level of the map.
-|feed.geomap.layer.geojson.url|null|no|List of GeoJSON layers to add to the map. Example: ["https://localhost:8000/service-area.geojson","https://localhost:8000/network-lines.geojson"]
+|feed.geomap.layer.geojson|null|no|List of GeoJSON layers to add to the map with optional style attribute. The style object can have styling properties for stroke, fill, image, and text styles as defined in OpenLayer flat style (https://openlayers.org/en/latest/apidoc/module-ol_style_flat.html). Example: [{"url":"https://localhost:8000/service-area.geojson"},{"url":"https://localhost:8000/network-lines.geojson", "style": {"stroke-color": "blue"}}]
```

Comment:
As an example : i would put 
```

[{"url":"assets/layers/service-area.geojson"},{"url":"https://localhost:8000/network-lines.geojson", "style": {"stroke-color": "blue"}}]
```

to show that is possible to set a local file instead of an url